tests/unit/test_steps.py

```diff
@@ -13,24 +13,28 @@
 from __future__ import absolute_import
 
 import pytest
+import boto3
+import os
 
 from stepfunctions.exceptions import DuplicateStatesInChain
-from stepfunctions.steps import Pass, Succeed, Fail, Wait, Choice, ChoiceRule, Parallel, Map, Task, Retry, Catch, Chain
+from stepfunctions.steps import Pass, Succeed, Fail, Wait, Choice, ChoiceRule, Parallel, Map, Task, Retry, Catch, Chain, \
+    utils
```

Comment:
Should we reorganize the imports so that each is on another line to make is more readable? It would also make it easier to spot the change next time a new import is added from stepfunctions.step.

Something like: 
```
from stepfunctions.steps import (
    Catch,
    Chain,
    Choice,
    ChoiceRule,
    Fail,
    Map,
    Parallel,
    Pass,
    Retry,
    Succeed,
    Task,
    Wait,
    utils,
)
```

Reply:
I was following the legacy across this repo and it has many places import like this. As these imports have relatively short name, to me importing same line is a better choice. Make the list longer is definitely clearer but the looking will be clumsy.
But let me know if this is hard to read. If so I can maybe divide them into 2 or 3 lines so that we have a clearer reading as well as keeping the context less bulky.

Reply:
This makes sense. So far I think there are definitely different style existing in this repo.